packages/libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -689,11 +708,10 @@ export class SolanaWeb3Manager {
     )
 
     const data = `${type}:${id}:${blocknumber}:${purchaserUserId}`
-
     const memoInstruction = new TransactionInstruction({
       keys: [
         {
-          pubkey: senderAccount,
+          pubkey: isSenderUserBank ? this.feePayerKey : senderAccount,
```

Comment:
in retrospect we don't actually need to sign this which might help clean this up:

> If zero accounts are provided to the signed-memo instruction, the program succeeds when the memo is valid UTF-8, and logs the memo to the transaction log.
>
> If one or more accounts are provided to the signed-memo instruction, all must be valid signers of the transaction for the instruction to succeed.
https://spl.solana.com/memo

Reply:
oh wuttt